api/v1/service/resource/resource.go

```diff
@@ -176,6 +178,52 @@ func (s *service) ByCatalogKindNameVersion(ctx context.Context, p *resource.ByCa
 	}
 }
 
+func (s *service) fetchCatalogClonePath(ctx context.Context, catalogName string) (string, error) {
+	baseClonePath := "/tmp/catalog"
+	catalog := model.Catalog{}
+	query := s.DB(ctx).Model(model.Catalog{}).Where("name = ?", strings.ToLower(catalogName))
+	if err := query.First(&catalog).Error; err != nil {
+		return "", err
+	}
+	u, _ := url.Parse(catalog.URL)
+	z := filepath.Join(baseClonePath, u.Host, u.Path+"@"+catalog.Revision)
+	return z, nil
+}
+
+func (s *service) ByCatalogKindNameVersionReadme(ctx context.Context,
+	p *resource.ByCatalogKindNameVersionReadmePayload) (*resource.ResourceVersionReadme, error) {
+	clonePath, err := s.fetchCatalogClonePath(ctx, p.Catalog)
+	if err != nil {
+		return nil, err
+	}
+	readmePath := fmt.Sprintf("%s/%s/%s/%s/README.md", clonePath, strings.ToLower(p.Kind), p.Name, p.Version)
```

Comment:
Should we change this to 
`readmePath := fmt.Sprintf("%s/%s/%s/%s", catalogPath, strings.ToLower(p.Kind), p.Name, p.Version`

readme := readmePath + "/README.md"

This would be more easy to read I think

Reply:
I don't have any strong preferences. I am fine with either